ui/minimal/README.md

```diff
@@ -0,0 +1,16 @@
+Minimal server driven user interface
```

Comment:
I'm not sure whether it's a good idea to introduce new UI for this, when it could be just added to the debug UI.